src/bin/server/server.rs

```diff
@@ -72,7 +75,12 @@ pub fn run_server(host: &str, port: &str, settings: &Settings) {
         let (rdr, wtr) = socket.split();
         let lines = lines(BufReader::new(rdr));
         let responses = lines.map(move |line| {
-            let resp = handler::gen_response(line.clone(), &mut state.borrow_mut());
+            #[cfg(feature="debug")]
```

Comment:
How does this feature work?  Is this meant to be for debugging during development rather than at runtime?

Reply:
During development. It's a compiler flag. https://doc.rust-lang.org/book/first-edition/conditional-compilation.html

Reply:
Yeah, I'm familiar with conditional compilation, I just wanted to know if it's something that we wanted to worry about in deployments for tracking down problems at runtime or it it was only useful during development.

Reply:
Probably a good idea to remove it TBH

Reply:
If it's useful during dev, I'm fine with keeping it in.  I was just trying to understand the reasoning behind adding it.

Reply:
The idea is to toggle a flag and have all sort of useful printout to debug with. Though chances are the next bug will be a lot more non-obvious so it's really not necessary.